### ISSUE TITLE
Issue while resolving versions of editable packages.  `uv pip install  -e {package}`

### ISSUE THREAD

**Issue by moaddib666** [astral-sh/uv#1476]
Steps to reproduce 
1. Install package A in editable mode
2. Install package B that relay on package A in editable mode

Expected result:
Packages A and B has been installed.

Actual result:
```
Audited 1 package in 3ms
uv pip install --prerelease=allow -e ecosystem/utils/ecosystem-utils-logging[test]
Audited 1 package in 4ms
uv pip install --prerelease=allow -e ecosystem/utils/ecosystem-utils-configuration[test]
   Built file:///Users/moaddib/PycharmProjects/spaceship/ecosystem.python/ecosystem/utils/ecosystem-utils-configuration                                                                                Built 1 editable in 829ms
  × No solution found when resolving dependencies:
  ╰─▶ Because ecosystem-utils-logging==0.0.1a0 was not found in the package registry and ecosystem-utils-configuration==0.0.1a0 depends on ecosystem-utils-logging==0.0.1a0, we can conclude that
      ecosystem-utils-configuration==0.0.1a0 cannot be used.
      And because you require ecosystem-utils-configuration==0.0.1a0, we can conclude that the requirements are unsatisfiable.
make: *** [install] Error 1
```

**Comment by charliermarsh** [comment]
This is the same as https://github.com/astral-sh/uv/issues/1661. (Gonna merge into that issue. This one was first, but I wrote up some notes on the other issue by chance.)

**Comment by zanieb** [comment]
Hi! This should be addressed in the latest release (0.1.27) via #2596.

Let us know if you have any problems.
